apps/condo/pages/ticket/index.tsx

```
@@ -304,13 +305,19 @@ const TicketTable = ({
                         <ActionBar
                             message={selectedTicketKeys.length > 0 && `${CountSelectedTicketLabel}: ${selectedTicketKeys.length}`}
                             actions={[
+                                <Button
+                                    key='createTicket'
+                                    type='primary'
+                                    children={CreateTicket}
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)